explore-analyze/query-filter/languages/esql-cross-clusters.md

```diff
@@ -470,10 +470,27 @@ FROM my-index-000001,cluster*:my-index-*,cluster_three:-my-index-000001
 
 
 ## Optional remote clusters [ccq-skip-unavailable-clusters]
+```{applies_to}
+stack: ga 9.1
+```
+
+If the remote cluster is configured with `skip_unavailable: true` (the default setting), the cluster would be set
+to `skipped` or `partial` status but the query will not fail, if:
+
+* The remote cluster is disconnected from the querying cluster, either before or during the query.
+* The remote cluster does not have the requested index.
+* An error happened while processing the query on the remote cluster.
+
+The `partial` status will be used if the remote query was partially successful and some data was returned.
```

Comment:
```suggestion
```

I suggested explaining skipped and partial in the opening section, so can remove this if accept that :)